src/views/SelectionChooser.ts

```diff
@@ -22,12 +29,14 @@ export default class SelectionChooser {
 
   private readonly target: IDType | null;
   private readonly readAble: IDType | null;
+  private readonly readableTargetIDType: IDType | null;
```

Comment:
this is because we can pass IDTypeLikes (= IDType | string) to the SelectionChooser and in the constructor the SelectionChooser `resolve`s the IDTypeLike to an actual IDType